powersimdata/input/tests/test_transform_profile.py

```diff
@@ -36,6 +34,11 @@ def get_plant_with_resource(base_grid, resource):
     return plant_id
 
 
+def _get_plant_ids_for_type(grid, type):
+    plant_ids = list(grid.plant.isin(profile_type[type]).query("type == True").index)
```

Comment:
I think `grid.plant.isin(profile_type[type]).query("type == True")` can be simplified to `grid.plant.query("type in @profile_type[@type]")`.

Reply:
Done, thanks!